install/010-fileserver.conf

```diff
@@ -1,21 +1,23 @@
 Listen 9999
 
 <VirtualHost 0.0.0.0:9999>
-    ServerName recodex-fileserver.localhost.localdomain
-    LogLevel info
+	ServerName recodex-fileserver.localhost.localdomain
+	LogLevel info
 
-    <Directory "/opt/recodex-fileserver">
-        Require all granted
-    </Directory>
-
-    <Location />
-	AuthName "ReCodEx fileserver"
-	AuthType Basic
-	AuthUserFile /etc/httpd/recodex_htpasswd
-	Require valid-user
-    </Location>
-
-    ProxyPass / http://127.0.0.1:10999/
-    ProxyPassReverse / http://127.0.0.1:10999/
+	WSGIDaemonProcess recodex-fileserver user=recodex group=recodex threads=5
```

Comment:
I'm no expert in Apache configuration, but I suggest using the `processes` option instead of `threads` and raising the limit so that we don't get drowned by requests that just download files. It is also possible to bypass the Python code for sending exercise files entirely using mod_rewrite.